docs/specification.md

```diff
@@ -763,7 +763,7 @@ Parameter (in Michelson):
 - Frozen tokens from voters and proposal submitter associated with those proposals
   are returned in the  form of tokens in governance token contract:
   - If proposal got rejected due to the quorum was not met or the quorum was met but upvotes are less then downvotes:
-    - The return amount for the proposer is equal to or less than the slashed amount based on `rejectedProposalReturnValue`.
+    - The return amount for the proposer is equal to or less than the slashed amount based on `rejectedProposalSlashValue`.
```

Comment:
```suggestion
    - The return amount for the proposer is equal to or less than the slashed amount based on `rejected_proposal_slash_value`.
```
let's use the name of the `lambda` in LIGO, after all the haskell name is only interesting for us and should not really be part of the spec